RELEASE.md

```diff
@@ -1,59 +0,0 @@
-# Spryker OS - DevVM - How to release
-
-## Manual release
```

Comment:
Where do these docs go?

Reply:
obsolete, 
will move to docs

Reply:
https://github.com/spryker/devvm/releases/tag/v4.1.0
and will add the docs one here after it's finalized

Reply:
Readme are never absolute and a must for all repos

Reply:
Please keep at least all basic info

Reply:
ok I will add that too

Reply:
Still feels like we lose a lot of meta info now with the almost empty readme.
Are we sure that is linked and otherwise documented somewhere else then to this extend?